tests/dummy_qrc/qblox.py

```diff
@@ -73,41 +73,32 @@ def create(runcard_path=RUNCARD):
 
     instruments = load_instrument_settings(runcard, instruments)
 
-    modules["qcm_bb0"] = qcm_bb0
-    modules["qcm_bb1"] = qcm_bb1
-    modules["qcm_rf0"] = qcm_rf0
-    modules["qcm_rf1"] = qcm_rf1
-    modules["qcm_rf2"] = qcm_rf2
-    modules["qrm_rf_a"] = qrm_rf_a
-    modules["qrm_rf_b"] = qrm_rf_b
+    modules = {
+        name: instrument
+        for name, instrument in instruments.items()
+        if isinstance(instrument, (ClusterQCM_RF, ClusterQCM_BB, ClusterQRM_RF))
+    }
```

Comment:
There is something weird happening with the `modules` here which also happens in qiboteam/qibolab_platforms_qrc#87. 

The `modules` variable is never used after it is defined as this dictionary. `modules` is used in one instance when you instantiate the `QbloxController`, however this comes before this definition and in fact you are passing an empty list of `modules` to the controller. Are you sure this is right?

Reply:
I'm taking care of it, thanks.

Reply:
In your opinion something like this can work as the final style for a qblox runcard?
```python
    modules = {
    "qcm_bb0": ClusterQCM_BB("qcm_bb0", f"{ADDRESS}:2", cluster),
    "qcm_bb1": ClusterQCM_BB("qcm_bb1", f"{ADDRESS}:4", cluster),
    "qcm_rf0": ClusterQCM_RF("qcm_rf0", f"{ADDRESS}:6", cluster),
    "qcm_rf1": ClusterQCM_RF("qcm_rf1", f"{ADDRESS}:8", cluster),
    "qcm_rf2": ClusterQCM_RF("qcm_rf2", f"{ADDRESS}:10", cluster),
    "qrm_rf_a": ClusterQRM_RF("qrm_rf_a", f"{ADDRESS}:16", cluster),
    "qrm_rf_b": ClusterQRM_RF("qrm_rf_b", f"{ADDRESS}:18", cluster)
    }

    controller = QbloxController("qblox_controller", cluster, modules)
    twpa_pump = SGS100A(name="twpa_pump", address="192.168.0.36")
    
    instruments = {
        controller.name: controller,
        twpa_pump.name: twpa_pump,
    }
    instruments.update(modules)
    instruments = load_instrument_settings(runcard, instruments)

    # Create channel objects
    channels = ChannelMap()
    # Readout
    channels |= Channel(name="L3-25_a", port=modules["qrm_rf_a"].ports["o1"])
    channels |= Channel(name="L3-25_b", port=modules["qrm_rf_b"].ports["o1"])
    # Feedback
    channels |= Channel(name="L2-5_a", port=modules["qrm_rf_a"].ports["i1"])
    channels |= Channel(name="L2-5_b", port=modules["qrm_rf_b"].ports["i1"])
    # Drive
    channels |= Channel(name="L3-15", port=modules["qcm_rf0"].ports["o1"])
    channels |= Channel(name="L3-11", port=modules["qcm_rf0"].ports["o2"])
    channels |= Channel(name="L3-12", port=modules["qcm_rf1"].ports["o1"])
    channels |= Channel(name="L3-13", port=modules["qcm_rf1"].ports["o2"])
    channels |= Channel(name="L3-14", port=modules["qcm_rf2"].ports["o1"])
    # Flux
    channels |= Channel(name="L4-5", port=modules["qcm_bb0"].ports["o1"])
    channels |= Channel(name="L4-1", port=modules["qcm_bb0"].ports["o2"])
    channels |= Channel(name="L4-2", port=modules["qcm_bb0"].ports["o3"])
    channels |= Channel(name="L4-3", port=modules["qcm_bb0"].ports["o4"])
    channels |= Channel(name="L4-4", port=modules["qcm_bb1"].ports["o1"])
    # TWPA
    channels |= Channel(name="L3-28", port=None)
    channels["L3-28"].local_oscillator = twpa_pump
```